### PR TITLE
feat: support svg devcards with an embedded png hack

### PR DESCRIPTION
**Pull request by kopancek**
Embed the existing png image in svg

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]

File: src/routes/devcards.ts [hunk 1]
```diff
@@ -93,6 +93,19 @@ export default async function (fastify: FastifyInstance): Promise<void> {
           return res.status(404).send();
         }
         const type = req.query?.type?.toLowerCase() ?? 'default';
+
+        // for svg, return the same image as png, wrapped in svg tag
+        if (format === 'svg') {
+          const svgString = `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><image xlink:href="${req.protocol}://${req.hostname}${req.originalUrl.replace('.svg', '.png')}" /></svg>`;
```
Comment (idoshamun) [inline src/routes/devcards.ts:99]: Does it include query params? I guess we should include them

Comment (kopancek) [reply src/routes/devcards.ts:99]: yes, it does

`originalUrl` unintuitively is everything behind the root path, so `/devcards/v2/.*` for this case.

[the remark this answers is not in the document]
Comment (idoshamun) [reply src/routes/devcards.ts:99]: Yes it's ideal! Good call

**Review by idoshamun — APPROVED** [review]
Awesome!

**Review by capJavert — APPROVED** [review]
Great idea 🚀

**Pull request merged**